### ISSUE TITLE
Overpass font not correctly aligned

### ISSUE THREAD

**Issue by JosuaMeier** [google/fonts#5795]
**Describe the bug**
In version 4.0.0 the overpass font is not vertically aligned center. 
![Bildschirmfoto 2023-01-16 um 14 30 32](https://user-images.githubusercontent.com/9655522/213180228-7ebaa0cd-19df-4b01-a24d-9e5289f8cc00.png)

Here is the correct version 3.0.5:
![Bildschirmfoto 2023-01-16 um 14 33 49](https://user-images.githubusercontent.com/9655522/213180351-e25bbefd-17bc-4528-a6dd-74c988a29877.png)

**To Reproduce**
https://jsbin.com/gotevucaku/edit?html,css,output

Correct with version 3.0.5 version:
https://jsbin.com/niwevodahe/1/edit?html,css,output

**Expected behavior**
Font should be vertically center aligned. (worked in version 3.x)

**Additional context**
Mac, Chrome

**Comment by JosuaMeier** [comment]
Any updates here?

**Comment by RosaWagner** [comment]
Hi @JosuaMeier, thanks for the report. Unfortunately we won't be able to fix that cause it would break websites served with this font. More info here: https://googlefonts.github.io/gf-guide/onboarding#font-upgrades

**Comment by chrissimpkins** [comment]
We're going to have a look at the diffenator comparisons between what the Red Hat team have in their main branch and what we ship to understand the impact on line spacing in order to achieve the improved vertical alignment with e.g., icons.

**Comment by tarjep** [comment]
What is the status on this? This misalignment is really annoying.

**Comment by thlinard** [comment]
Issue upstream: https://github.com/RedHatOfficial/Overpass/issues/91

**Comment by thlinard** [comment]
Btw, in version 4, `typoDescender` and `hheaDescender` have been set to `-756`,  which is equal to the value of yMin. This value is the value of  ‎`| 007C VERTICAL LINE`, and even though this matches what is done in the first example of [Google Fonts documentation | Vertical metrics](https://googlefonts.github.io/gf-guide/metrics.html), it was not relevant to do so in this case.

The conversion of old values (Overpass Heavy v3.0.5: `yMin= -378`, `hheaLineGap = 266`, `typoDescender= -250` and `hheaDescender= -284`, Use_Typo_Metrics enabled in fsSelection) ​​​​that was done when switching to variable format would deserve to be reviewed.

**Comment by emmamarichal** [comment]
Hi everyone, he you can find the diff between the last version and the version currently served:

https://github.com/user-attachments/assets/226777ce-f57c-4a8e-b677-9754866bd9c6

imo, the diff is much too big to take the risk of breaking website layouts. Overpass is served a lot.
I understand the non-centered caps are annoying, but unfortunately I don't think we can do anything here, except serving locally the version you want. 
What do you think @davelab6?

<img width="1269" height="462" alt="Image" src="https://github.com/user-attachments/assets/8b5e8250-edd3-4cb9-8fb4-68fb2ce30952" />

**Comment by armandabric** [comment]
Is serving the new/fixed version on another name is something conceivable?

**Comment by ecmahutte** [comment]
Hi everyone,

At our company we use Overpass as our primary brand font. I’d like to add my support for a version update or a separate 'UI' release of Overpass on Google Fonts.

We are currently experiencing significant issues with the vertical alignment (baseline) in the version served by Google Fonts (v4.000). To make it work on our website, we’ve had to implement CSS ascent-override and descent-override fixes just to get buttons looking balanced.

The biggest pain point however is currently in Figma and using it in design systems and new concepts. Because the metrics are off, we are forced to use 'asymmetric y-padding' for components like buttons and labels to make them appear centered. This makes our design hand-off and system maintenance incredibly cumbersome.

I’ve compared the Google Fonts version (v4.000) with the latest version from Delve Fonts (v4.020; FEAKit 1.0). The newer version seems to address some of these issues, but since Figma pulls from the Google Fonts library, we are stuck with the 'broken' metrics.

I understand the concern regarding layout shifts for existing websites. Therefore, I strongly support the proposal to offer a corrected version under a new name (e.g., 'Overpass UI') if a direct update is not possible. This would allow us to use a technically sound version for modern UI work without breaking legacy web content.

Looking forward to a potential update!

**Comment by emmamarichal** [comment]
Hi everyone, hi @ecmahutte,

Thank you for the detailed write-up and for sharing your use case.

Unfortunately, we're not able to onboard a near-duplicate of an existing font where the only difference lies in the metrics. As much as we understand the frustration, accepting this would set a precedent that would be difficult to manage: many fonts on Google Fonts have similar vertical metrics issues, and opening the door to duplicated versions for each of them would make the library increasingly complex to maintain.

That said, here are two practical paths forward:

1. **Download the latest version locally:** As you noted, Delve Fonts' v4.020 seems to already address some of these issues. Using it locally in Figma would give you the corrected metrics without waiting on Google Fonts.

2. **A UI variant in the upstream repo:** I can generate a UI version of Overpass with corrected metrics and propose it to the upstream repo. We could add a small note in the "About" section on the Google Fonts page pointing users to this variant in the upstream repo, so it remains discoverable for anyone who needs it for UI work.

I hope one of these works for your team. Let me know if you'd like me to move forward with the upstream proposal!

**Comment by ecmahutte** [comment]
Hi @emmamarichal,

Thank you for the response and for offering to move forward with an upstream proposal.

I want to correct something from my earlier comment: after comparing the TTF files directly, I believe that the vertical metrics in v4.020 (Delve Fonts) are identical to v4.000 (Google Fonts). So unfortunately, downloading the latest version locally does not resolve the metrics issue, which means option 1 is off the table.

To test whether your option 2 approach would work in practice, I generated a modified version of the Delve v4.020 TTF with adjusted vertical metrics — derived from the ascent-override and descent-override values we currently apply in CSS on our website — and renamed it 'Overpass UI'. Essentially, I wanted to verify whether a separately named variant would solve the Figma issue before you invest time in building a proper version.

It does work: because the name is different, Figma treats it as a distinct font family, which means it does not conflict with the Google Fonts version of Overpass and can be selected explicitly in a design system. This confirms that your option 2, if made available for local install under a separate name, would be a valid solution for teams like ours.

That said, the metrics I used are a rough approximation based on our existing CSS fix, not derived from the actual glyph data. So I would not suggest using my version as the basis for an official variant — that is exactly the kind of thing better left to someone with your expertise.

If you are able to create a proper corrected version and make it available — whether via the upstream repo or elsewhere — that would of course be the better solution, and we could switch to that version for both our Figma setup and our website.

One thing I would strongly recommend (from my use case) regardless of how it is distributed: giving the corrected version a distinct name, such as 'Overpass UI'. The reason is practical: applications like Figma pull from the Google Fonts library by default, and the priority behaviour between an online font and a locally installed font with the same name is not reliable or transparent to designers. A different name makes it unambiguous which version is in use, which matters a lot for design system consistency across a team.

Happy to share the TTF I generated as a stopgap if that is useful for anyone in this thread.

**Comment by emmamarichal** [comment]
Hi @ecmahutte,

Okay, great, yes, that solution seems perfectly feasible. And no worries, I was planning to name it "UI" to avoid any confusion or conflicts locally during font installation.

I should be able to patch these new vertical metrics values ​​and the new name directly onto the binaries.
I'll let you know here once it's done. Thank you!

**Comment by emmamarichal** [comment]
Hello!
Here’s my PR updating the v-metrics by creating a new UI version. I can’t guarantee that Red Hat will merge my PR, but at least you can use the versions available on my fork.

https://github.com/RedHatOfficial/Overpass/pull/107
